beancount_import/source/schwab_csv.py

```diff
@@ -923,49 +1114,100 @@ def _load_transactions(filename: str) -> List[RawEntry]:
         assert match, title
         account = match.groupdict()["account"]
         reader = csv.DictReader(csvfile)
-        assert reader.fieldnames == expected_field_names, reader.fieldnames
-        found_total_line = False
-        for lno, row in enumerate(reader):
-            # Final row in CSV is not a real transaction
-            if row["Date"] == "Transactions Total":
-                found_total_line = True
-                continue
-            # If there are no transactions, Schwab includes a row with just ""
-            if row["Date"] == "":
-                continue
-            assert not found_total_line
-            date = _convert_date(row["Date"])
-            action = SchwabAction(row["Action"])
-            symbol = STRIP_FROM_SYMBOL_RE.sub("", row["Symbol"])
-            description = row["Description"]
-            quantity = int(row["Quantity"]) if row["Quantity"] else None
-            price = _convert_decimal(row["Price"])
-            fees = _convert_decimal(row["Fees & Comm"])
-            amount = _convert_decimal(row["Amount"])
-            if OPTION_RE.match(row["Symbol"]) and quantity:
-                # this is an option, sold in lots of 100
-                quantity *= 100
-            entries.append(
-                RawEntry(
-                    account=account,
-                    date=date,
-                    action=action,
-                    symbol=symbol,
-                    description=description,
-                    quantity=quantity,
-                    price=price,
-                    fees=fees,
-                    amount=Amount(amount, currency="USD") if amount else None,
-                    filename=filename,
-                    line=lno + 2,
-                )
-            )
+        if reader.fieldnames == expected_brokerage_field_names:
+            entries = _load_brokerage_transactions(reader, account, filename)
+        elif reader.fieldnames == expected_banking_field_names:
+            entries = _load_banking_transactions(reader, account, filename)
+        else:
+            raise RuntimeError(f"Unexpected header {reader.fieldnames}")
     entries.reverse()
     return entries
 
 
+def _load_banking_transactions(reader: csv.DictReader, account: str, filename):
+    entries = []
+    transaction_start_line = 2
```

Comment:
It looks like all the source line numbers in the metadata for the generated transactions in the test are off by one (too high). Maybe this just needs to be `transaction_start_line = 1`?